src/stackdriver_agent.c

```diff
@@ -189,6 +189,35 @@ static int sagt_read(user_data_t *user_data)
     }
     meta_data_destroy(md);
   }
+  // Metric point_count from write_gcm.c.
```

Comment:
[optional] nit - leave a empty line above the comment to be consistent with other blocks of code.